dir.targets

```diff
@@ -28,6 +28,15 @@
     <TargetFrameworkProfile></TargetFrameworkProfile>
     <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
     <PortableNuGetMode>true</PortableNuGetMode>
+
```

Comment:
I think this change should be removed from this PR now with the current changes you have. @sokket can pick this part of with his work.